packages/client/src/views/SysAdmin/Performance/Vsexports/VSExport.tsx

```diff
@@ -0,0 +1,180 @@
+  }
+`
+type VSExportProps = {
+  items: VsExport[]
+}
+
+const notificationActionButtonHandler = () => {
+  window.location.reload()
+}
+
+function downloadURI(uri: string, name: string) {
+  const link = document.createElement('a')
+  link.href = uri
+  link.download = name
+  document.body.appendChild(link)
+  link.click()
+  document.body.removeChild(link)
+}
+
+function TabContent(props: VSExportProps) {
+  const intl = useIntl()
+  const items: VsExport[] = props.items
+  return (
+    <>
+      {items.map((item: VsExport) => {
+        const fileName = intl.formatMessage(messages.vitalStatisticsExport, {
+          year: item.year,
+          event: item.event,
+          fileSize: ''
+        })
+        const sizeValue = intl.formatMessage(messages.vitalStatisticsExport, {
+          year: item.year,
+          event: item.event,
+          fileSize: item.fileSize
+        })
+
+        return (
+          <ListViewSimplified key={`${item.createdOn}_${item.event}`}>
+            <ListViewItemSimplified
+              label={<Label id={`${item.year}_label`}>{item.year}</Label>}
+              value={<Value id={`${item.createdOn}_value`}>{sizeValue}</Value>}
+              actions={
+                <DynamicHeightLinkButton
+                  id={item.url}
+                  disabled={false}
+                  onClick={() =>
+                    downloadURI(`${MINIO_URL}${item.url}`, fileName)
+                  }
+                >
+                  {messages.export.defaultMessage}
+                </DynamicHeightLinkButton>
+              }
+            />
+          </ListViewSimplified>
+        )
+      })}
+    </>
+  )
+}
+
+const VSExport = () => {
+  const intl = useIntl()
+
+  const [activeTabId, setActiveTabId] = React.useState(Event.Birth)
+  const tabSections = [
+    {
+      id: Event.Birth,
+      title: intl.formatMessage(messages.birthTabTitleExport)
+    },
+    {
+      id: Event.Death,
+      title: intl.formatMessage(messages.deathTabTitleExport)
+    }
+  ]
+
+  // TODO: Replace with Frame component */
+
+  return (
+    <>
+      <SysAdminContentWrapper
+        id="vsexport-wrapper"
+        hideBackground={true}
+        isCertificatesConfigPage={true}
+      >
+        <UserTable id="vsexport_list">
+          <Content
+            title={intl.formatMessage(messages.vsexport)}
+            titleColor={'copy'}
+            subtitle={intl.formatMessage(messages.vsEmptyStateText)}
+            tabBarContent={
+              <FormTabs
+                sections={tabSections}
+                activeTabId={activeTabId}
+                onTabClick={(id) => setActiveTabId(id)}
+              />
+            }
+          >
+            <Query<GetVsExportsQuery>
+              query={GET_TOTAL_VSEXPORT}
+              fetchPolicy={'no-cache'}
+            >
+              {({ data, loading, error }) => {
+                if (error) {
+                  return (
+                    <ErrorToastNotification
```

Comment:
Let's use <Toast /> (type error) here from the component library. ErrorToastNotification is depreciated